### ISSUE TITLE
config's include property name different from documentation

### ISSUE THREAD

**Issue by rjoaopereira** [amzn/style-dictionary#214]
Hello,

The config object's `include` property name is not coherent with the one present in the online docs:

https://amzn.github.io/style-dictionary/#/config
https://github.com/amzn/style-dictionary/blob/master/lib/extend.js#L103

One is plural and the other is not.

**Comment by dbanksdesign** [comment]
Good catch! Thanks for bringing this to our attention. We recently have been working to improve our documentation and must have missed this. I created a PR for this: #217

**Comment by dbanksdesign** [comment]
Documentation has been updated: https://amzn.github.io/style-dictionary/#/config
